openandroidinstaller/assets/configs/sweet.yaml

```
@@ -0,0 +1,57 @@
+metadata:
+  maintainer: BjornW
+  brand: xiaomi
+  device_name: Xiaomi Redmi Note 10 Pro
```

```suggestion
  device_name: Xiaomi Redmi Note 10 Pro
  brand: xiaomi
```
Just a line order change as documented [here](https://github.com/openandroidinstaller-dev/openandroidinstaller/blob/main/docs/how_to_contribute_your_own_installation_configurations.md#how-to-write-metadata)